### PR TITLE
Added several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@peternewman ready for review

**Review by bl-ue — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4136,6 +4139,8 @@ brillant->brilliant
 brimestone->brimstone
 bringin->bringing
 bringtofont->bringtofront
+brite->bright
+briter->brighter
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:4143]: Add `briten` and variants?
```suggestion
brite->bright
briten->brighten
britened->brightened
britener->brightener
briteners->brighteners
britenes->brightenes
britening->brightening
briter->brighter
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -24799,6 +24811,8 @@ rquiring->requiring
 rranslation->translation
 rranslations->translations
 rrase->erase
+rrror->error
+rrrors->errors
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:24815]: Add `rrrored` and `rrroring`?
```suggestion
rrror->error
rrrored->errored
rrroring->erroring
rrrors->errors
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -27490,6 +27504,11 @@ supproting->supporting
 supprots->supports
 supprt->support
 supprted->supported
+suppurt->support
+suppurted->supported
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:27508]: Add `suppurter` and `suppurters`?
```suggestion
suppurt->support
suppurted->supported
suppurter->supporter
suppurters->supporters
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -28981,6 +29000,7 @@ triked->tricked
 trikery->trickery
 triks->tricks, trikes,
 triky->tricky
+trilinar->trilinear
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:29003]: Suggest `trillionaire` & add `trilinars`?
```suggestion
trilinar->trilinear, trillionaire,
trilinar->trillionaire
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Comment by bl-ue** [comment]
ping @luzpaz did this get lost?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -4857,6 +4868,10 @@ challege->challenge
 chambre->chamber
 chambres->chambers
 Champange->Champagne
+chanage->change
+chanaged->changed
```
Comment (bl-ue) [inline codespell_lib/data/dictionary.txt:4872]:
```suggestion
chanaged->changed
chanager->changer
```

**Review by bl-ue — APPROVED** [review]

**Pull request merged**